pallets/did/src/lib.rs

```diff
@@ -433,16 +629,38 @@ impl<T: Config> Pallet<T> {
 		let did_entry: DidDetails =
 			<Did<T>>::get(op.get_did()).ok_or(DidError::StorageError(StorageError::DidNotPresent))?;
 
+		Self::verify_operation_signature_for_entry(op, signature, &did_entry)?;
+
+		Ok(())
+	}
+
+	/// Verify the signature of a generic DidOperation, and
```

Comment:
The first sentence of this comment seems off. I would remove the part about the return.

```suggestion
	/// Verify the signature of a generic DidOperation.
```